index.js

```diff
@@ -103,17 +103,21 @@ function check(port, host) {
     }
 
     function onErrorCb(err) {
-        if (err.code !== 'ECONNREFUSED') {
-            //debug('check - promise rejected, error: '+err.message);
-            deferred.reject(err);
-        } else {
-            //debug('ECONNREFUSED');
-            inUse = false;
-            //debug('check - promise resolved - not in use');
-            deferred.resolve(inUse);
-        }
-        cleanUp();
-    }
+       console.log('ERROR CODE:::', err.code);
```

Comment:
please only print to console in exceptional cases (e.g. unexpected error code), if at all

Reply:
this section would be more readable as a switch() statement